public/pages/EditFeatures/components/AggregationSelector/__tests__/AggregationSelectior.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
small - typo in filename

Reply:
Somehow I put a duplicate wrong file. Will remove.